src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -40,27 +49,51 @@
 
     @Test
     public void parse_allFieldsPresent_success() {
```

Comment:
why you need to change `AMY` and `BOB` huh?

Reply:
AMY is a doctor, while BOB is a patient. BOB has one more age field, which affects how the fields are parsed.